app/models/user.rb

```diff
@@ -113,8 +113,8 @@ class User < ApplicationRecord
     joins(:comments).where("comments.commentable": commentables).distinct
   end
   scope :by_username_email_or_document_number, ->(search_string) do
-    string = "%#{search_string}%"
-    where("username ILIKE ? OR email ILIKE ? OR document_number ILIKE ?", string, string, string)
+    search = "%#{search_string.strip}%"
+    where("username ILIKE :search OR email ILIKE :search OR document_number ILIKE :search", search: search)
```

Comment:
Since we leave the `?` style in the `.search` method, how about using the `?` style here as well? I'm suggesting this style because it's the one used everywhere else; I don't have a strong prefence for one style or the other, and when in doubt I prefer consistency.